build.config

```diff
@@ -1,4 +1,4 @@
-email=suvft@rkpvgr.pbz
+email=onypxubyr@abjurer
```

Comment:
I think you want to revert that unless you want to get an email every time I test the build 😉